2021/42xxx/CVE-2021-42013.json

```diff
@@ -1,4 +1,51 @@
 {
```

Comment:
('Expecting property name enclosed in double quotes: line 2 column 1 (char 2)',)